### PR TITLE
Fixing export error message

### PR DESCRIPTION
**Pull request by latran**
The export error was changed due to a PR from katello #[27838](https://github.com/Katello/hammer-cli-katello/pull/697).  Updating the test case to incorporate the new error message.

### CODE REVIEW

**Comment by latran** [comment]
Test result of two test cases that uses the error message.
```
$ pytest tests/foreman/cli/test_satellitesync.py::ContentViewSync::test_negative_export_cv_with_background_policy_repo
================================= test session starts ================
platform linux -- Python 3.6.8, pytest-4.6.3, py-1.8.0, pluggy-0.12.0
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/ltran/Projects/robottelo
plugins: services-1.3.1, mock-1.10.4
collecting ... 2019-12-11 20:59:36 - conftest - DEBUG - Collected 1 test cases
2019-12-11 15:59:36 - robottelo.ssh - DEBUG - Instantiated Paramiko client 0x7f17a93da438
2019-12-11 15:59:36 - robottelo.ssh - INFO - Connected to [ibm-x3250m4-03.lab.eng.rdu2.redhat.com]
2019-12-11 15:59:36 - robottelo.ssh - INFO - >>> rpm -q satellite
2019-12-11 15:59:37 - robottelo.ssh - INFO - <<< stdout
satellite-6.7.0-4.beta.el7sat.noarch

2019-12-11 15:59:37 - robottelo.ssh - DEBUG - Destroyed Paramiko client 0x7f17a93da438
2019-12-11 15:59:37 - robottelo.host_info - DEBUG - Host Satellite version: 6.7
2019-12-11 15:59:37 - robottelo.helpers - DEBUG - Generated file bz_cache.json with BZ collect data
collected 1 item                                                                                                                                                                                                                                                           

tests/foreman/cli/test_satellitesync.py .                                                                                                                                                                                                                            [100%]

======================== 1 passed in 168.87 seconds ======================
$ pytest tests/foreman/cli/test_satellitesync.py::ContentViewSync::test_negative_export_cv_with_on_demand_repo
============================== test session starts ===============
platform linux -- Python 3.6.8, pytest-4.6.3, py-1.8.0, pluggy-0.12.0
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/ltran/Projects/robottelo
plugins: services-1.3.1, mock-1.10.4
collecting ... 2019-12-11 21:03:21 - conftest - DEBUG - Collected 1 test cases
2019-12-11 16:03:21 - robottelo.ssh - DEBUG - Instantiated Paramiko client 0x7f1fcedc34a8
2019-12-11 16:03:21 - robottelo.ssh - INFO - Connected to [ibm-x3250m4-03.lab.eng.rdu2.redhat.com]
2019-12-11 16:03:21 - robottelo.ssh - INFO - >>> rpm -q satellite
2019-12-11 16:03:23 - robottelo.ssh - INFO - <<< stdout
satellite-6.7.0-4.beta.el7sat.noarch

2019-12-11 16:03:23 - robottelo.ssh - DEBUG - Destroyed Paramiko client 0x7f1fcedc34a8
2019-12-11 16:03:23 - robottelo.host_info - DEBUG - Host Satellite version: 6.7
2019-12-11 16:03:23 - robottelo.helpers - DEBUG - Generated file bz_cache.json with BZ collect data
collected 1 item                                                                                                                                                                                                                                                           

tests/foreman/cli/test_satellitesync.py .                                                                                                                                                                                                                            [100%]
================ 1 passed in 172.96 seconds =========================
```

File: tests/foreman/cli/test_satellitesync.py [hunk 1]
```diff
@@ -981,7 +981,8 @@ def test_negative_export_cv_with_on_demand_repo(self):
             })
         self.assert_error_msg(
             error,
-            'All exported repositories must be set to an immediate download policy and re-synced'
+            "Could not export the content view:\n  Error: Ensure the content view version"
+            " {} has at least one repository.\n".format(cv_dict.values())[1]
```
Comment (mirekdlugosz) [inline tests/foreman/cli/test_satellitesync.py]: Are you sure about this indexing? It takes second character from created string. It makes more sense to me to take second value from dict, but you also need to wrap `cv_dict.values()` in `list()` for that.

```python
>>> cv_dict = {'one': 1, 'two': 2}
>>> "Could not export the content view:\n  Error: Ensure the content view version {} has at least one repository.\n".format(cv_dict.values())[1]
'o'
>>> "Could not export the content view:\n  Error: Ensure the content view version {} has at least one repository.\n".format(cv_dict.values()[1])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'dict_values' object does not support indexing
>>> "Could not export the content view:\n  Error: Ensure the content view version {} has at least one repository.\n".format(list(cv_dict.values())[1])
'Could not export the content view:\n  Error: Ensure the content view version 2 has at least one repository.\n'
```

**Comment by latran** [comment]
```
$ pytest tests/foreman/cli/test_satellitesync.py::ContentViewSync::test_negative_export_cv_with_background_policy_repo
=============================== test session starts ===================
platform linux -- Python 3.6.8, pytest-4.6.3, py-1.8.0, pluggy-0.12.0
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/ltran/Projects/robottelo
plugins: services-1.3.1, mock-1.10.4
collecting ... 2019-12-12 18:34:56 - conftest - DEBUG - Collected 1 test cases
2019-12-12 13:34:56 - robottelo.ssh - DEBUG - Instantiated Paramiko client 0x7f7f345aa4e0
2019-12-12 13:34:56 - robottelo.ssh - INFO - Connected to [ibm-x3250m4-03.lab.eng.rdu2.redhat.com]
2019-12-12 13:34:56 - robottelo.ssh - INFO - >>> rpm -q satellite
2019-12-12 13:34:57 - robottelo.ssh - INFO - <<< stdout
satellite-6.7.0-4.beta.el7sat.noarch

2019-12-12 13:34:57 - robottelo.ssh - DEBUG - Destroyed Paramiko client 0x7f7f345aa4e0
2019-12-12 13:34:57 - robottelo.host_info - DEBUG - Host Satellite version: 6.7
2019-12-12 13:34:57 - robottelo.helpers - DEBUG - Generated file bz_cache.json with BZ collect data
collected 1 item                                                                                                                                                                                                                                                           

tests/foreman/cli/test_satellitesync.py .                                                                                                                                                                                                                            [100%]

========================= 1 passed in 173.97 seconds ===============
$ pytest tests/foreman/cli/test_satellitesync.py::ContentViewSync::test_negative_export_cv_with_on_demand_repo
======================= test session starts ========================
platform linux -- Python 3.6.8, pytest-4.6.3, py-1.8.0, pluggy-0.12.0
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/ltran/Projects/robottelo
plugins: services-1.3.1, mock-1.10.4
collecting ... 2019-12-12 18:39:19 - conftest - DEBUG - Collected 1 test cases
2019-12-12 13:39:19 - robottelo.ssh - DEBUG - Instantiated Paramiko client 0x7fe44ffc5400
2019-12-12 13:39:19 - robottelo.ssh - INFO - Connected to [ibm-x3250m4-03.lab.eng.rdu2.redhat.com]
2019-12-12 13:39:19 - robottelo.ssh - INFO - >>> rpm -q satellite
2019-12-12 13:39:20 - robottelo.ssh - INFO - <<< stdout
satellite-6.7.0-4.beta.el7sat.noarch

2019-12-12 13:39:20 - robottelo.ssh - DEBUG - Destroyed Paramiko client 0x7fe44ffc5400
2019-12-12 13:39:20 - robottelo.host_info - DEBUG - Host Satellite version: 6.7
2019-12-12 13:39:20 - robottelo.helpers - DEBUG - Generated file bz_cache.json with BZ collect data
collected 1 item                                                                                                                                                                                                                                                           

tests/foreman/cli/test_satellitesync.py .                                                                                                                                                                                                                            [100%]

=================== 1 passed in 178.97 seconds ========
```

File: tests/foreman/cli/test_satellitesync.py (hunk 1 above)
Comment (latran) [reply tests/foreman/cli/test_satellitesync.py]: @mirekdlugosz, I see what you mean and have updated the code.  So this should be up to standard now.

**Pull request merged**